apps/web/src/pages/TalentNominations/NominationGroup/components/NominationGroupSidebar.tsx

```diff
@@ -0,0 +1,365 @@
+} from "@gc-digital-talent/ui";
+import {
+  FragmentType,
+  getFragment,
+  graphql,
+  TalentNominationGroupStatus,
+} from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { insertBetween } from "@gc-digital-talent/helpers";
+
+import { getFullNameLabel } from "~/utils/nameUtils";
+import { getClassificationName } from "~/utils/poolUtils";
+
+import NominatedForList from "./NominatedForList";
+
+type AccordionStates = "nominee-contact-information" | "comments" | "";
+
+export const NominationGroupSidebar_Fragment = graphql(/* GraphQL */ `
+  fragment NominationGroupSidebar on TalentNominationGroup {
+    id
+    ...NominationGroupSidebarForList
+    status {
+      value
+      label {
+        localized
+      }
+    }
+    nominee {
+      id
+      email
+      workEmail
+      firstName
+      lastName
+      role
+      telephone
+      preferredLang {
+        label {
+          localized
+        }
+      }
+      currentClassification {
+        id
+        group
+        level
+      }
+      department {
+        name {
+          localized
+        }
+      }
+    }
+    nominations {
+      id
+      nominator {
+        id
+        firstName
+        lastName
+      }
+    }
+  }
+`);
+
+export interface NominationGroupSidebarProps {
+  talentNominationGroupQuery: FragmentType<
+    typeof NominationGroupSidebar_Fragment
+  >;
+}
+
+const NominationGroupSidebar = ({
+  talentNominationGroupQuery,
+}: NominationGroupSidebarProps) => {
+  const intl = useIntl();
+
+  const [accordionState, setAccordionState] = useState<AccordionStates>("");
+
+  const talentNominationGroup = getFragment(
+    NominationGroupSidebar_Fragment,
+    talentNominationGroupQuery,
+  );
+
+  const nominations = talentNominationGroup.nominations ?? [];
+
+  // dialog to go somewhere in span?
+  const nominationsSortedByNominator = nominations.sort((a, b) => {
+    return (
+      (a.nominator?.lastName ?? "").localeCompare(
+        b.nominator?.lastName ?? "",
+      ) ||
+      (a.nominator?.firstName ?? "").localeCompare(b.nominator?.firstName ?? "")
+    );
+  });
+  const nominatorsList = nominationsSortedByNominator.map((nomination) => (
+    <span key={nomination.id}>
+      {getFullNameLabel(
+        nomination.nominator?.firstName,
+        nomination.nominator?.lastName,
+        intl,
+      )}
+    </span>
+  ));
+  const nominatorListCommaSeparated = insertBetween(
+    // eslint-disable-next-line formatjs/no-literal-string-in-jsx
+    <span>, </span>,
+    nominatorsList,
+  );
+
+  // set the styling colours of the status bar and button
+  const statusColours = {
+    textBox: "base(success.lightest) base:dark(success.lightest)",
+    border: "base(success.darker) base:dark(success.darker)",
+    text: "base(success.darker) base:dark(success.darker)",
+    buttonBox: "base(success.darker) base:dark(success.darker)",
+    button: "base(success.lightest) base:dark(success.lightest)",
```

Comment:
I don't get why the button colouring is not working out right 😕 

![image](https://github.com/user-attachments/assets/65c41ee7-5ba2-42ba-8481-68d761bfd277)

With this sample colour change though 

![image](https://github.com/user-attachments/assets/50e46e4d-c9d0-4cd4-899d-ef37210e1c64)

Something happens...

![image](https://github.com/user-attachments/assets/1a7b1133-6b00-4e22-b55a-3f96a630d306)

Reply:
@yonikid15 will help with this.

Reply:
Thanks!